examples/ssd1306_example/src/ssd1306_example.erl

```diff
@@ -19,34 +19,44 @@
 -export([start/0]).
 
 start() ->
+    % This is a button on ESP-WROOM-32 development board
+    Pin = 0,
+    GPIO = gpio:open(),
+    gpio:set_direction(GPIO, Pin, input),
+    ok = gpio:set_int(GPIO, Pin, both),
+
     SSD1306Config = #{
         sda_pin => 21,
         scl_pin => 22
     },
     {ok, SSD1306} = ssd1306:start(SSD1306Config),
     ssd1306:clear(SSD1306),
-    ssd1306:set_contrast(SSD1306, 0),
-    loop(SSD1306, text).
+    ssd1306:set_contrast(SSD1306, 255),
+
+    loop(SSD1306, Pin, 0).
 
-loop(SSD1306, text) ->
+loop(SSD1306, Pin, Touches) ->
+    {{Year,Month,Day},{Hour,Min,Sec}} = erlang:localtime(),
     Text = io_lib:format(
-        "  !!AtomVM!!~n~nFree heap:~n~p bytes",
-        [erlang:system_info(esp32_free_heap_size)]
+        "  !!AtomVM!!~n~nFree heap:~n~p bytes~n~p:~p:~p ~nBOOT:~p ~nTouches: ~p ",
+        [erlang:system_info(esp32_free_heap_size),
+        Hour,Min,Sec,
+        gpio:digital_read(Pin),
+        Touches
+        ]
     ),
     io:format("Displaying text: ~s~n", [Text]),
     ssd1306:clear(SSD1306),
     ssd1306:set_text(SSD1306, Text),
-    timer:sleep(5000),
-    loop(SSD1306, text);
-%% TODO
-loop(SSD1306, qrcode) ->
```

Comment:
I think instead of removing the qrcode example by overwriting it, I think it would be preferable to add another separate example for the new functions.

Reply:
Thanks for pointing that out. Sorry for missing it.

I've re-arranged the changes so it does not damage the other examples.